### PR TITLE
Contact section

### PR DESCRIPTION
**Pull request by stephaniesaylor**
Updated Help with new Contact section.

### CODE REVIEW

File: src/scripts/components/articles/Center_listings_create.html [hunk 1]
```diff
@@ -1,216 +1,217 @@
...
+<b>Screenshots</b>
+<br />Houses the images that appear in the Overview tab on the listing's Detailed View.
+<br />You can add infinite screenshots. The system asks for a:
+	<ul>
+		<li>Preview Image<br />
+		<i>Appears on the Overview tab</i></li>
+		<li>Full Size Image<br />
+		<i>Opens outside the Overview tab after a user clicks the Preview Image</li>
+	</ul>
+	</i>
```
Comment (wski) [inline src/scripts/components/articles/Center_listings_create.html]: This is weird, Should the `</i>` be inside of the `</li>` ?

File: src/scripts/components/articles/GlobalToolbar_overview.html [hunk 2]
```diff
@@ -34,13 +34,13 @@
   part: 'global',
   article: 'Webtop_overview',
   title: 'Understanding Webtop'
-}}>Webtop</Link> your customizable workspace within OZONE.</li>
+}}>Webtop</Link>, your customizable workspace within OZONE.</li>
```
Comment (wski) [inline src/scripts/components/articles/GlobalToolbar_overview.html]: Not sure we need this comma grammatically

File: src/scripts/components/articles/Center_listings_create.html (hunk 1 above)
Comment (stephaniesaylor) [inline src/scripts/components/articles/Center_listings_create.html]: updated

File: src/scripts/components/articles/GlobalToolbar_overview.html (hunk 2 above)
Comment (stephaniesaylor) [inline src/scripts/components/articles/GlobalToolbar_overview.html]: The comma is fine. No change.

**Comment by bhalt** [comment]
Ready for merge @wski :+1:

**Pull request merged**